app/model/lib/submission_process.py

```diff
@@ -70,6 +69,8 @@ def persist_submission_to_database(submission_form):
             _create_average_measurements(db_trans_session, study, experiment)
 
         submission_form.save()
+        submission_form.save_backup(study_id=study.publicId, project_id=project.publicId)
```

Comment:
We now have submission backups, so in case of issues, we can go back to them in the admin, copy-paste their study design and/or take their file, and repair a particular study.